src/featureslist.h

```diff
@@ -164,49 +139,27 @@ FEATURE_SIMPLE(mobileOnboarding,     // Feature ID
                QStringList(),        // feature dependencies
                FeatureCallback_iosOrAndroid)
 
-FEATURE(
-    multiAccountContainers,      // Feature ID
-    "Multi-Account Containers",  // Feature name
-    true,                        // Is Major Feature
-    L18nStrings::WhatsNewReleaseNotesMultiAccountContainersHeader,  // Display
-                                                                    // name
-    L18nStrings::
-        WhatsNewReleaseNotesMultiAccountContainersShortDescription,  // Description
-    L18nStrings::
-        WhatsNewReleaseNotesMultiAccountContainersDescription,  // LongDescr
-    "qrc:/ui/resources/features/"
-    "multi-account-containers-preview.png",  // ImagePath
-    "qrc:/ui/resources/features/multi-account-containers-icon.svg",  // IconPath
-    "https://support.mozilla.org/kb/"
-    "protect-your-container-tabs-mozilla-vpn?utm_source=mozilla.vpn."
-    "client&utm_medium=in-app-link&utm_content=whats-new-panel&utm_"
-    "campaign=vpn-better-together",  // link URL
-    "2.7",
-    false,          // Can be flipped on
-    false,          // Can be flipped off
-    QStringList(),  // feature dependencies
-    FeatureCallback_multiAccountContainers)
+FEATURE_SIMPLE(multiAccountContainers,      // Feature ID
```

Comment:
You can remove this feature. And also this https://searchfox.org/mozilla-vpn-client/source/src/featureslistcallback.h#85